docs/advanced-customization/config/email-server-settings.md

```diff
@@ -1,17 +1,25 @@
 ---
-description: Configure your Forem's SMTP server settings.
-position: 16
+description: Configure your Forem's Email Server Settings.
+position: 5
 ---
 
-# SMTP Settings
+# Email Server Settings (SMTP)
+
+
+## Forem Cloud Clients
+
+As a Forem Cloud client, we provide an email server managed by the Forem team. All settings are managed by us and the from and reply email addresses for all emails are set as noreply@forem.com.
```

Comment:
We've added the ability for Forem Cloud Clients to override their Sendgrid settings and use custom SMTP settings if they wish to.